selfdrive/ui/paint.cc

```diff
@@ -307,7 +307,7 @@ static void ui_draw_driver_view(UIState *s) {
   const int face_size = 85;
   const int icon_x = is_rhd ? rect.right() - face_size - bdr_s * 2 : rect.x + face_size + bdr_s * 2;
   const int icon_y = rect.bottom() - face_size - bdr_s * 2.5;
-  ui_draw_circle_image(s, icon_x, icon_y, face_size, "driver_face", face_detected);
+  ui_draw_circle_image(s, icon_x, icon_y, face_size, "driver_face", s->scene.dmonitoring_state.getIsActiveMode());
```

Comment:
in driverview the box surrounding the face will still be drawn regardless

Reply:
sounds good

Reply:
hmm, actually let's make a TODO and give some indicator of whether it's active in driver view

Reply:
the icon would tell the activeness in driverview also